src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propsList = sourceString.split(';');
+
+  return propsList.reduce((propsObject, property) => {
+    const currentProp = property.split(':');
+
+    if (currentProp[0] && currentProp[1]) {
+      propsObject[currentProp[0].trim()] = currentProp[1].trim();
```

Comment:
Ok, but better to avoid incoming object mutation (now we know, that it is not used anywhere outside this method, but still it's a bad practice)

Reply:
You mean this object - propsObject ?